src/index.html

```diff
@@ -1,12 +1,325 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <a class="logo" href="#">
+        <img src="./images/logo.png" alt="Store Logo">
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__list">
```

Comment:
I'm not sure that is good class name for element
each tag `li` is item, not a list

you can use class `nav__list` for `ul` tag